airflow/dag_processing/processor.py

```diff
@@ -305,7 +306,7 @@ def is_ready(self) -> bool:
 
     @property
     def start_time(self) -> float:
-        return self._process.create_time()
+        return timezone.utcnow().timestamp()
```

Comment:
You've probably found the right root cause, but returning now for start_time isn't the fix I don't think.

Reply:
Updated